charts/jenkins/values.yaml

```diff
@@ -169,6 +169,12 @@ controller:
       # It delays the initial readyness probe while Jenkins is starting
       # initialDelaySeconds: 60
 
+  # PodDisruptionBudget config
+  podDisruptionBudget:
+    enabled: false
+    annotations: {}
+    labels: {}
```

Comment:
Unset options should be added below here commented out so people reading values.yaml can see it

Reply:
Sure... Added in [`f2e05bb` (#461)](https://github.com/jenkinsci/helm-charts/pull/461/commits/f2e05bb9eddf7b0327e18475007ab13a59e4ddfa)